code_schemes/yes_no.json

```diff
@@ -0,0 +1,91 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Yes_No",
```

Comment:
Which question is this a yes/no response to? Please include this in the scheme name.

Reply:
Have renamed to Radio Questions Yes_No. The scheme is shared across the Yes_No radio questions.